crates/lib3h/src/transport/transport_multiplex/route.rs

```diff
@@ -0,0 +1,240 @@
+use crate::transport::{error::*, protocol::*};
+use detach::prelude::*;
+use lib3h_ghost_actor::prelude::*;
+use url::Url;
+
+use super::LocalRouteSpec;
+
+enum RouteToParentContext {}
+
+enum RouteToInnerContext {
+    AwaitBind(
+        GhostMessage<RequestToChild, RequestToParent, RequestToChildResponse, TransportError>,
+    ),
+    AwaitSend(
+        GhostMessage<RequestToChild, RequestToParent, RequestToChildResponse, TransportError>,
+    ),
+}
+
+pub struct TransportMultiplexRoute {
```

Comment:
`route_spec` member is not used so its hard to see what this struct is for since it just forwards messages up & down.
Can't we just give the space gateway the `endpoint_parent` created during `get_agent_space_route()` ?

Reply:
Ha! that is indeed how this ended up working out. ... I'm a little hesitant to remove it just in case we end up needing to put some additional functionality in here along the lines of what I was originally thinking before I removed it all...

But... I guess it's not that hard to resurrect from git history.

Reply:
done